MaintenanceManager/MaintenanceManager.cpp

```diff
@@ -374,6 +374,11 @@ namespace WPEFramework {
             /* we moved every thing to a thread */
             /* only when dcm is getting a DCM_SUCCESS/DCM_ERROR we say
              * Maintenance is started until then we say MAITENANCE_IDLE */
+
```

Comment:
I don't see a need for this. If the global/class variables are properly protected and notifications are sent appropriately.